build.gradle

```diff
@@ -3,26 +3,329 @@ buildscript {
+            pom.withXml {
+                Node root = asNode()
+                root.appendNode('name', 'realm-annotations')
+                root.appendNode('description', 'Annotations for Realm: a replacement for SQLite & ORMs.')
+                root.appendNode('url', 'http://realm.io')
+                root.appendNode('packaging', 'jar')
+                root.children().last() + commonPom
+            }
+        }
+        annotationsProcessor(MavenPublication) {
+            artifactId = 'realm-annotations-processor'
+            artifact file("${rootDir}/realm/realm-annotations-processor/build/libs/realm-annotations-processor-${currentVersion}.jar")
+            pom.withXml {
+                Node root = asNode()
+                root.appendNode('name', 'realm-annotations-processor')
+                root.appendNode('description', 'Annotations processor for Realm: a replacement for SQLite & ORMs.')
+                root.appendNode('url', 'http://realm.io')
+                root.appendNode('packaging', 'jar')
+                root.children().last() + commonPom
+            }
+        }
+        gradlePlugin(MavenPublication) {
+            artifactId = 'gradle-realm-plugin'
+            artifact file("${rootDir}/gradle-plugin/build/libs/gradle-plugin-${currentVersion}.jar")
+            pom.withXml {
+                Node root = asNode()
+                root.appendNode('name', 'gradle-realm-plugin')
+                root.appendNode('description', 'Gradle plugin for Realm: a replacement for SQLite & ORMs.')
+                root.appendNode('url', 'http://realm.io')
+                root.appendNode('packaging', 'jar')
+                root.children().last() + commonPom
+            }
+        }
+    }
+}
+
+artifactory {
+    contextUrl = 'https://oss.jfrog.org/artifactory'
+    publish {
+        repository {
+            repoKey = 'oss-snapshot-local'
+            username = project.hasProperty('bintrayUser') ? bintrayUser : 'noUser'
+            password = project.hasProperty('bintrayKey') ? bintrayKey : 'noKey'
+        }
+        defaults {
+            publications ('library', 'annotations', 'annotationsProcessor', 'gradlePlugin')
+        }
+    }
+}
+
+bintray {
+    user = project.hasProperty('bintrayUser') ? bintrayUser : 'noUser'
+    key = project.hasProperty('bintrayKey') ? bintrayKey : 'noKey'
+
+    dryRun = true
+    publish = false
+
+    publications = ['library', 'annotations', 'annotationsProcessor', 'gradlePlugin']
+    pkg {
+        repo = 'maven'
+        name = 'realm-android'
+        desc = 'Realm for Android'
+        websiteUrl = 'http://realm.io'
+        issueTrackerUrl = 'https://github.com/realm/realm-java/issues'
+        vcsUrl = 'https://github.com/realm/realm-java.git'
+        licenses = ['Apache-2.0']
+        labels = ['aar', 'android', 'realm']
+        publicDownloadNumbers = false
+    }
 }
 
-task clean(type: Delete) {
-    delete rootProject.buildDir
+release {
```

Comment:
It is probably my Gradle foo that is failing me here, but this doesn't seem to be called from anywhere?